tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+                    # the radius.
+                    labeldistance=1.125,
+                    # The labeldistance keyword argument specifies the float as
+                    # a percentage of the radius to place labels.
+                    wedgeprops=dict(width=0.3),
+                    # The wedgeprops keyword argument can also take
+                    # dictionaries to pass on to the artists. In this
+                    # instance, the float for width sets the wedge size as a
+                    # percentage of the radius starting from the outer edge.
+                    shadow=True)
+
+wedges, texts, autotexts = budget_pie
+# The pie() method unpacks into three objects, wedges, texts, and autotexts.
+# These Artists have their own methods for addtional customization.
+
+ax.legend(wedges,
+          # The wedges variable unpacked from the method serve as the handles
+          # for the legend.
+          categories,
+          # The information from the data categories correspond to each
+          # respective wedge instead of redundant labeling from the previous
+          # pie chart.
+          title='Categories',
+          # The legend has a title keyword argument.
+          bbox_to_anchor=(0.125, 0.5),
+          # This keyword argument in conjunction with loc places the legend
+          # at a specific point. The tuple floats are coordinates for the
+          # Figure.
+          loc='center right'
+          # The loc keyword argument works in conjunction with bbox_to_anchor
+          # and in this instance, determines the part of the legend for
+          # placement. Without bbox_to_anchor, Matplotlib automatically
+          # manages coordinates in relation to specifications of the
+          # parameters of loc.
+          )
+
+ax.set_title('Average Monthly Income Expenses')
+ax.axis('equal')
+
+fig.tight_layout()
+# The Figure method tight_layout() manages the space between all Artists to
+# maximize visiblity on the Figure. This method also contains various
+# parameters for configuration.
+
+plt.show()
+
+##############################################################################
+#
+#
+#
+# Multiple Graphs within a Figure
+# -------------------------------
+#
+# For multiple graphs using a single Figure, explicit and implicit approaches
+# use a similar convention for mapping out multiple Axes. Matplotlib manages
+# more than one Axes in a two-dimensional matrix. They are arranged by row
+# amount and then by column amount. The third argument represents the specific
+# Axes involved.
+#
+# When looking for more complex solutions to multiple graphs within a Figure,
+# use the :class:`matplotlib.gridspec.GridSpec` module to organize the layout.
+#
+# Explicit
+# ^^^^^^^^
+
+# Explicit with OOP
+
+fig, (ax1, ax2) = plt.subplots(1, 2,
+                               sharey='row',
+                               figsize=[8, 4],
+                               constrained_layout=True)
```

Comment:
wonder if would be easier to follow using the semantic layout tool: https://matplotlib.org/devdocs/tutorials/provisional/mosaic.html#sphx-glr-tutorials-provisional-mosaic-py